src/middleware/index.ts

```diff
@@ -226,8 +227,22 @@ const validFreeText = (freeText: string) => {
   return false;
 };
 
+const VALID_SORT_FIELDS = ["startDate", "endDate", "id", "title", "eo:cloud_cover"];
+
+const validSortBy = (sortBy: string | string[] | SortObject[]) => {
+  const fields: string[] = parseSortFields(sortBy);
+
+  return fields.every((value) => {
+    const isDescending = value.startsWith("-");
+    const cleanSortBy = isDescending ? value.slice(1) : value;
+    const fieldName = cleanSortBy.replace(/^properties\./, "");
```

Comment:
```suggestion
    // Allow for `properties` prefix
    const fieldName = cleanSortBy.replace(/^properties\./, "");
```
ignore if you think this is obvious to anyone working in cmr-stac in the future